app/src/test/java/com/example/todolist/DetailedListViewModelUnitTest.java

```diff
@@ -8,36 +8,38 @@
 import com.example.todolist.dto.ListItem;
 import com.example.todolist.dto.ToDoList;
 import com.example.todolist.repositories.listItemsRepository.IListItemRepository;
-import com.example.todolist.repositories.listRepository.IListRepository;
 import com.example.todolist.ui.detailScreen.DetailedListViewModel;
 import static org.mockito.Mockito.*;
 
 import java.time.ZonedDateTime;
 import java.time.temporal.ChronoUnit;
 import java.util.ArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class DetailedListViewModelUnitTest {
     private final int testId = 1;
     private final int checked = 1;
     private final int unchecked = 0;
+    private final int threads = 4;
 
     private IListItemRepository mockedListItemRepository;
-    private IListRepository mockedListRepository;
     private DetailedListViewModel sut;
+    private final ExecutorService executorService = Executors.newFixedThreadPool(threads);
```

Comment:
Почему ты решил использовать общий `ThreadPool` для всех тестов в этом файле?

Reply:
I've considered this solution the right one, because the app works the same way. `ViewModel` receives `ThreadPool` from `ToDoListApp` class once, in its constructor. So, I've thought that single `ThreadPool` for all tests is suitable here

Reply:
Есть ли шанс, что этот объект продолжит выполнение кода из другого теста?

Reply:
Да, такой шанс есть, об этом я не подумал

Reply:
Тогда что нужно сделать?

Reply:
Каждый раз перед тестом (в сетапе) передавать новый пул